addons/controller_icons/ControllerIcons.gd

```diff
@@ -13,7 +13,7 @@ var _custom_input_actions := {}
 
 var _last_input_type = InputType.KEYBOARD_MOUSE
 var _settings
-var _file := File.new()
+var _file : FileAccess
```

Comment:
Thanks to file access now being static, this isn't needed anymore, so this line can be erased, and any references to `_file` changed to `FileAccess` instead

Reply:
This should be fixed now in d3db640456b85db9460a7525ce0de0898e3c73e1